bootstrap/mod/fabric/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+    application
+}
+
+architectury {
+    platformSetupLoomIde()
+    fabric()
+}
+
+dependencies {
+    modImplementation(libs.fabric.loader)
+    modApi(libs.fabric.api)
+
+    api(project(":mod", configuration = "namedElements"))
+    shadow(project(path = ":mod", configuration = "transformProductionFabric")) {
+        isTransitive = false
+    }
+    shadow(projects.core) {
+        exclude(group = "com.google.guava", module = "guava")
+        exclude(group = "com.google.code.gson", module = "gson")
+        exclude(group = "org.slf4j")
+        exclude(group = "com.nukkitx.fastutil")
+        exclude(group = "io.netty.incubator")
+    }
+
+    // This should be in the libs TOML, but something about modImplementation AND include just doesn't work
+    include(modImplementation("me.lucko", "fabric-permissions-api", "0.2-SNAPSHOT"))
```

Comment:
Maybe try getting the individual components directly (group, id, version) instead of just, e.g. libs.fabric.permissions.api?